corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -4,11 +4,7 @@
 {% load compress %}
 
 {% block title %}
-    {% if request.couch_user.is_web_user %}
-        {% trans "Preview Form" %}
```

Comment:
since web user's can now use cloudcare just like a mobile worker, i think it makes sense to remove this